tests/repr_eq_test.py

```
@@ -29,6 +29,9 @@
 from fastkml.enums import PairKey
 from tests.base import Lxml
 from tests.base import StdLibrary
+from fastkml.kml import KML, Document, Placemark  
```

Ruff

Redefinition of unused `Point` from line 24

(F811)